### DISCUSSION TITLE
Erase SRAM

### DISCUSSION THREAD

**Discussion by Chomemel** [sanni/cartreader#501] [Ideas & Suggestions]
Hi Sanni,
I don’t know if it is possible or if it would be hard for you to do it, but I wonder if it would be possible to add an « SRAM erase » option. 
I sometimes do cartmods for translations (mostly SNES). Most of the time, saves from the new game overwrite saves from the old game without any problems. But sometimes there are problems (game freezing) and I have to disconnect the battery to erase the SRAM to get it work.
So, erasing the SRAM without using the iron would be really convienient.

**Comment by Icesythe7** [comment]
It is def possible but all it does is write 0xFF to every memory location...you can get an app like HXD and just create a save file with all FF and just upload it to the cart which would be effectively the same thing while you wait, alternatively if you are unsure of the size simply dump your current save and open it in HXD and ctrl + a then right-click > fill selection > FF > ok > save > upload back to cart

**Comment by Mythris29** [comment]
Would love to see this added as a feature but in the meantime thank you for the instructions on creating a blank file @Icesythe7 I was asked to do that to help troubleshoot and wasn't sure how to do it :)
